news/2016-10-01-tip1-logging.asciidoc

```diff
@@ -63,10 +63,18 @@ Weld log messages are grouped into several categories:
 * `org.jboss.weld.Bean`
 * `org.jboss.weld.Context`
 * `org.jboss.weld.El`
-* `org.jboss.weld.Conversation`
```

Comment:
One last thing, what did you remove `org.jboss.weld.Conversation`?
To make it shorter? Or did you mean to replace it with the Config one?